src/holofuel/pages/TransactionHistory/TransactionHistory.js

```diff
@@ -43,10 +43,17 @@ function usePollCompletedTransactions ({ since }) {
 const FILTER_TYPES = ['all', 'withdrawals', 'deposits', 'pending']
 
 export default function TransactionsHistory ({ history: { push } }) {
-  const { loading: ledgerLoading, data: { holofuelLedger: { balance: holofuelBalance } = {} } = {} } = useQuery(HolofuelLedgerQuery, { fetchPolicy: 'network-only' })
+  const { loading: ledgerLoading, data: { holofuelLedger: { balance: holofuelBalance } = {} } = {} } = useQuery(HolofuelLedgerQuery, { fetchPolicy: 'cache-and-network', pollInterval: 5000 })
   const { loading: loadingPendingTransactions, data: { holofuelWaitingTransactions = [] } = {} } = useQuery(HolofuelWaitingTransactionsQuery, { fetchPolicy: 'cache-and-network' })
   const { loading: loadingCompletedTransactions, data: { holofuelCompletedTransactions = [] } = {} } = useQuery(HolofuelCompletedTransactionsQuery, { fetchPolicy: 'cache-and-network' })
 
+  const [hasloadedFirstLedger, setHasloadedFirstLedger] = useState()
+  useEffect(() => {
+    if (holofuelBalance) {
+      setHasloadedFirstLedger(true)
+    }
+  }, [holofuelBalance, setHasloadedFirstLedger])
```

Comment:
we don't need this, can just use `!holofuelBalance` instead  of creating `hasloadedFirstLedger `